ruby_review_week_four.rb

```diff
@@ -2,36 +2,104 @@
 
 # 1. What is a variable?
 
+A variable is a value that is interchangeable
+
 # 2. What Ruby method can you use to make a loop from an array?
 
+to make a loop from an array you can use the 
+.each method 
+
 # 3. In Ruby, what are booleans?
 
+booleans are variables that equate to true or false
+
+
 # 4. How do classes relate to objects?
 
+objects are the parents of classes
+
 # 5. What is an instance variable?
 
+instance variable is a variable that is defined within a class
+
 # 6. Choose an everyday item (cup, computer, person) and give it two instance variables (which will be attributes), and one method (to describe a function of the object). Code the initialize method in full (for these attributes only), and the definition line of the instance method (but do not code the entire method).
 
+class Cup
+attr_reader :size, :color
+
+class Computer
+attr_reader :processor, :model
+
 # 7. What is handy about using gems in Ruby?
 
+using gems in Ruby is handy because it allows you to display your code with more functionality present
+
 # 8. Define a class called fish, and leave it empty.
+class Fish 
+end
 
 # 9. How would you use that class to make three fish objects?
+class Fish
+end
+
+fish1 = Fish.new
+fish2 = Fish.new
+fish3 = Fish.new
 
 # 10. Add two attributes to your fish class, name and species, and make them accessible for reading and writing.
 
+class Fish
+def initialize(name, species)
+    @name = name
+    @species = species
```

Comment:
nice! to make readable and writable you need to use the `attr_accessor` method call